components/connectivity-adapter/internal/appregistry/service/converter.go

```diff
@@ -1,9 +1,18 @@
 package service
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/kyma-incubator/compass/components/connectivity-adapter/internal/appregistry/model"
 	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
-	"github.com/kyma-incubator/compass/tests/director/pkg/ptr"
+)
+
+const (
+	// fields that exist in Application Registry but don't exist in Director GraphQL API, are stored in labels with special prefix
+	prefixUnmappedFields          = "compatibility_unmapped_fields_"
+	unmappedFieldIdentifier       = prefixUnmappedFields + "identifier"
```

Comment:
Did we discuss our convention of naming label keys? I remember something like `prefix/name`, but I don't know whether this is our agreed approach or not. Anyway, this is just an open question.

Reply:
changed